deepchem/models/tests/test_openvino.py

```diff
@@ -4,10 +4,53 @@
 import torch
```

Comment:
Please modify `.github/workflows/build.yml` and `.github/workflows/main.yml` to run the tests:
```
  push: # ci work when pushing master branch
    branches:
      - master
  pull_request: # ci work when creating a PR to master branch
    branches:
      - openvino
```